docs/content/posts/2025-02-TBD-release-1.20.1.md

```diff
@@ -0,0 +1,213 @@
+---
+title:  "Apache Flink 1.20.1 Release Announcement"
+date: "2025-02-TBDT00:00:00.000Z"
+aliases:
+- /news/2025/02/TBD/release-1.20.1.html
+authors:
+- afedulov:
+  name: "Alexander Fedulov"
+
+---
+
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.20 series.
```

Comment:
nit: isn't this the first bug fix release of the Flink 1.20 series?

Reply:
That is a good catch, it will be. :) I guess 1.19 copy/paste result.